src/ui/pages/ConnectionDetails/ConnectionDetails.test.tsx

```diff
@@ -704,256 +719,42 @@ describe("Checking the Connection Details Page when different Credentials are is
-
-    await waitFor(() => {
-      expect(getByTestId("connection-detail-date")).toBeVisible();
-      expect(getByTestId("connection-detail-date")).toHaveTextContent(
-        `${formatShortDate(
-          connectionsFix[2].connectionDate
-        )} - ${formatTimeToSec(connectionsFix[2].connectionDate)}`
-      );
-    });
-
-    await waitFor(() =>
-      expect(getByText("Received \"Access Pass Credential\"")).toBeVisible()
-    );
-
-    await waitFor(() =>
-      expect(getByTestId("connection-history-timestamp")).toHaveTextContent(
-        `${formatShortDate("2024-02-15T10:16:08.756Z")} - ${formatTimeToSec(
-          "2024-02-15T10:16:08.756Z"
-        )}`
-      )
-    );
-  });
-
-  test("We can see the connection details for PermanentResidentCard", async () => {
-    jest
-      .spyOn(Agent.agent.connections, "getConnectionById")
-      .mockResolvedValue(connectionsFix[1]);
-
-    jest
-      .spyOn(Agent.agent.connections, "getConnectionHistoryById")
-      .mockResolvedValue([
-        {
-          type: 0,
-          timestamp: "2024-02-13T10:16:26.919Z",
-          credentialType: "PermanentResidentCard",
-        },
-      ]);
-
-    const storeMocked = {
-      ...mockStore(initialStateFull),
-      dispatch: dispatchMock,
-    };
-    const { getByTestId, queryByTestId, getByText } = render(
-      <MemoryRouter initialEntries={[TabsRoutePath.CREDENTIALS]}>
-        <Provider store={storeMocked}>
-          <Route
-            path={TabsRoutePath.CREDENTIALS}
-            component={Creds}
-          />
-
-          <Route
-            path={RoutePath.CONNECTION_DETAILS}
-            component={ConnectionDetails}
-          />
-        </Provider>
-      </MemoryRouter>
-    );
-
-    act(() => {
-      fireEvent.click(getByTestId("connections-button"));
-    });
-
-    await waitFor(() => {
-      expect(queryByTestId("connection-item-0")).toBeNull();
-    });
-
-    expect(getByText(connectionsFix[1].label)).toBeVisible();
-
-    act(() => {
-      fireEvent.click(getByText(connectionsFix[1].label));
-    });
-
-    await waitFor(() =>
-      expect(getByText("Connected with \"Passport Office\"")).toBeVisible()
+      expect(getByText("13/02/2024 - 10:16:08")).toBeVisible()
     );
 
-    await waitFor(() => {
-      expect(getByTestId("connection-detail-date")).toBeVisible();
-      expect(getByTestId("connection-detail-date")).toHaveTextContent(
-        `${formatShortDate(
-          connectionsFix[1].connectionDate
-        )} - ${formatTimeToSec(connectionsFix[1].connectionDate)}`
-      );
-    });
-
     await waitFor(() =>
       expect(getByText("Received \"Permanent Resident Card\"")).toBeVisible()
     );
 
     await waitFor(() =>
-      expect(getByTestId("connection-history-timestamp")).toHaveTextContent(
-        `${formatShortDate("2024-02-13T10:16:26.919Z")} - ${formatTimeToSec(
-          "2024-02-13T10:16:26.919Z"
-        )}`
-      )
+      expect(getByText("14/02/2024 - 10:16:26")).toBeVisible()
```

Comment:
Because in source code you are using `formatShortDate` and `formatTimeToSec` to format date time. But both 2 functions are using `Date(date).toLocaleTimeString` and return value base on local timezone of user computer.

And this is the reason when I run this test case, it failed (because I live in vietnam). I think we are using CI/CD infra on your timezone but if we change it to other timezone this test maybe failed. Can you using `formatShortDate` and `formatTimeToSec` to format date time instead fixed value?